archive/d/dart/prime-number.dart

```diff
@@ -0,0 +1,38 @@
+// Issue 4970
+import 'dart.math';
+
+void main(List<String> args){
+  const String error_message= "Usage: please input a non-negative integer";
+  if (args.isEmpty){
+    print(error_message);
+    return;
+  }
+
+  try{
+    bool is_prime = true;
+    int num_needed = int.parse(args[0]);
+
+    if (num_needed.isNegative){
+      print(error_message);
+      return;
+    }
+
+    // If number is even other than 2; number is multiple of 5 other than 5; if it is 0; it is 1
+    if( (num_needed == 0)|| (num_needed == 1) || (num_needed %2 ==0 && num)needed != 2) || (num_needed %2 ==0 && num)needed != 2) ){
```

Comment:
This comment is incorrect. This has nothing to do with multiples of 5. Also, this `if` statement seems incorrect.